app/components/meta-authors/model.js

```diff
@@ -6,15 +6,16 @@ module.exports.save = (ref, data) => {
   data.authors = data.authors || [];
   // Normalize "authors" value; if saved from a Kiln form, it will be of the form
   // [{text: string}].
-  data.authors = data.authors
-    .map(author => typeof author === 'string' ? author : _get(author, 'text', ''));
+  data.authors = data.authors.map(author =>
```

Comment:
Could this `data.authors.map` be combined with the below on line 19?

Reply:
I believe they serve different purposes cause one is for the `save` method and the other for the `render` method